cmd/project/get.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2018, Google, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package project
+
+import (
+	"fmt"
+	"github.com/spinnaker/spin/util"
+	"net/http"
+
+	"github.com/spf13/cobra"
+	"github.com/spinnaker/spin/cmd/gateclient"
+)
+
+type GetOptions struct {
+	*projectOptions
+	expand bool
+}
+
+var (
+	getProjectShort   = "Get the specified project"
+	getProjectLong    = "Get the specified project"
+	getProjectExample = "usage: spin project get [options] project-name"
+)
+
+func NewGetCmd(prjOptions projectOptions) *cobra.Command {
+	options := GetOptions{
+		projectOptions: &prjOptions,
+		expand: false,
+	}
+
+	cmd := &cobra.Command{
+		Use:     "get",
```

Comment:
There's a specific endpoint to get a project [here](https://github.com/spinnaker/gate/blob/master/gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/ProjectController.groovy#L39), so I think this needs reworked into something more appropriate with what the command is doing, e.g. `spin project get-pipelines`.